velox/experimental/cudf/exec/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+add_library(
```

Comment:
What are the plans for cudf in velox? Do you want it to be part of libvelox.a? Or should it be something like `libvelox_cudf.a` as a separate component?

Reply:
Hmm. I am not sure. libcudf is a pretty large library, maybe 0.5 GB depending on how it's compiled. Do you think a separate component is appropriate? What factors besides binary size should play a role in this decision?

Reply:
I'm a build novice so excuse my misunderstanding but can't we have a libvelox_cudf.a that doesn't include libcudf. We'll just have to put `cudf` in `PUBLIC`. 

Related: how gluten is planning to use it: https://github.com/apache/incubator-gluten/blob/42014f14cca9a7281b856cde94806f80ac129fcc/cpp/velox/CMakeLists.txt#L343-L350

Reply:
As disscussed in the call let's stick with add_library for now